docs/development/build-mac.md

```diff
@@ -16,6 +16,8 @@ You must install Xcode.  This will allow you to build LiveCode for:
 You will not be able to compile the OS X desktop version of LiveCode
 unless you install some older OS X SDKs; see the next section for details.
 
+You will need to setup your Apple ID (aka developer account) in Xcode by going to _Preferences &rarr; Accounts_. without it the code signing step of the build process will fail. 
```

Comment:
@soapdog could you hard wrap this at 80 columns

Reply:
@montegoulding,
Sorry, I missed that comment. I just wrapped the text to 80 chars but there was one shell command that had 82 chars in it and I thought it was better to leave it that way than wrap.